src/operator/radio_types.rs

```diff
@@ -10,7 +13,7 @@ use serde::{Deserialize, Serialize};
 // EIP-712 name will change encoding, append other unique message types/names to decode properly
 #[derive(Eip712, EthAbiType, Clone, Message, Serialize, Deserialize, PartialEq, SimpleObject)]
 #[eip712(
-    name = "Graphcast POI Radio",
+    name = "Graphcast Listener Radio",
```

Comment:
iirc EIP712 name for message types must stay consistent between radios, otherwise they won't parse correctly

Reply:
Made it consistent with the naming of other message types, lmk what you think